front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -109,6 +109,7 @@ function CreateShipmentView() {
   const currentBase = allAcceptedTransferAgreements?.data?.base;
   const currentOrganisationLabel = `${currentBase?.organisation?.name} - ${currentBase?.name}`;
   const currentOrganisationId = globalPreferences.organisation?.id;
+  const currentOrganisationBases = globalPreferences.availableBases;
```

Comment:
available bases only shows the other bases in your organisation if you are a Headof Ops user. We will need to query the bases to create shipments with differently. @pylipp any suggestion how best to query all bases in your organisations even the ones you don not have access to?
Maybe:
```
organisation(id:2) {
      id
      name
      bases {
        id
        name
      }
    }
    ```

Reply:
@HaGuesto that's exactly the way to go.

Reply:
@pylipp I have only the name and id of other bases is available through this query. It is the same query used to create transferAgreements, right?